code/game/objects/items/stacks/medical.dm

```diff
@@ -2,15 +2,15 @@
 	name = "medical pack"
 	singular_name = "medical pack"
 	icon = 'icons/obj/stack/items.dmi'
-	amount = 5
-	max_amount = 5
+	amount = 10
+	max_amount = 10
```

Comment:
Any particular reasons why you doubled the uses of all advanced kits without mentioning it in the PR?

Reply:
that was the default value, i have not been able to find the code to change that my self. I told kaz about that, and since this requires them to do the perk side of the coding it's been left to them.

Reply:
It's not default value, since it was clearly changed as indicated by github. Furthermore, ATKs and ABKs on live only have five uses.

And FURTHERmore, here is the current code from master that is used by the server.

Reply:
![image](https://user-images.githubusercontent.com/65850818/94663095-912a7580-0309-11eb-87bd-6684a9ef628c.png)

Reply:
I know they only have five uses as I said I don't know how to change that specifically.  The values being off probably is from me coding this on something down stream and porting it over if i had to guess.

Reply:
Well, it's written right there, you only need to set 
`amount = 10` to 5 and same for `max_amount`

Reply:
Right done then. Ty.